wiki/Glossary/en.md

```diff
@@ -34,65 +34,62 @@ Authentic is a word that denotes a beatmap that emulates in every aspect of a sp
 
 Most beatmap packs are sets of beatmaps that were ranked during the same time period. However, in some packs may contain songs of a similar style or theme.
 
-Downloading beatmap packs is the recommended way of downloading beatmaps because this method of downloading beatmaps would save peppy bandwidth.
+To see the beatmap packs, see [Beatmap Pack Listing](https://osu.ppy.sh/p/packlist)
 
-To see the beatmap packs, see [Beatmap Pack Listing](http://osu.ppy.sh/p/packlist)
-
-### Beat Spacing
+### Beat spacing
 
 Beat spacing is a principle which states that the spacing of every object (except for spinners) should be either closer, if closely placed on the timeline, or farther, if placed farther apart on the timeline.
-This basically means, "close in time = close in space" and "farther in time = farther in space".
+This basically means, `close in time = close in space` and `farther in time = farther in space`.
 
-Your beat spacing plays an crucial role when the player is playing your beatmap!
-Consistent beat spacing is a basic requirement to rank a beatmap.
-This provides the player with a more intuitive experience as the player will learn where to move their cursor along the screen,  at a constant speed.
+Your beat spacing plays a crucial role when the player is playing your beatmap!
+Consistent beat spacing is a basic requirement to rank a beatmap because it provides the player with a more intuitive experience as the player will learn where to move their cursor along the screen, at a constant speed.
 
 Placing an object closer than the beat spacing may trick the player into clicking on them too soon.
 On the otherhand, placing them too far may force the player to move their cursor across the screen to click on them in time.
 A common exception to beat spacing is stacked hitcircles.
 
-To ensure consistent beat spacing, turn on the "Distance Snap" button on the right-hand side of the editor.
-You can choose a beat spacing value by holding [Alt] and sliding the selector at the top right until you reach the desired value.
+To ensure consistent beat spacing, turn on the `Distance Snap` button on the right-hand side of the editor.
+You can choose a beat spacing value by holding `ALT` and sliding the selector at the top right until you reach the desired value.
 Note that the higher the value would allow a more difficult beatmap.
 
 ### Beatmap
 
-A beatmap (or "level" or "map") is where the majority of osu! lies.
+A beatmap (also known as a level or a map) is where the majority of osu! lies.
 It is quite like the stages in the DS games.
 At their most basic form, beatmaps have a song, hitcircles, sliders, and/or spinners placed based on the rhythm of the music.
 Nowadays, beatmaps may also have a background image or video, or even a storyboard.
```

Comment:
Remove the link for BSS (below this paragraph). It links to elsewhere in the article, but that link won't even work